pkgs/applications/misc/minergate/default.nix

```diff
@@ -1,16 +1,16 @@
 { fetchurl, lib, stdenv, dpkg, makeWrapper, fontconfig, freetype, openssl, xorg, xkeyboard_config }:
 
 stdenv.mkDerivation {
-  version = "8.1";
+  version = "8.3";
   pname = "minergate";
   src = fetchurl {
     url = "https://minergate.com/download/ubuntu";
```

Comment:
Can you upload it to web.archive.org?

Reply:
does think [link](https://minergate.com/download/ubuntu) not work for you?

Reply:
This url describes a 'latest version', which means that this package will break as soon as minergate updates. We need a url with immutable contents.

Reply:
is this fine?

Reply:
when visiting the site itself i do not see the link provided here. maybe this specific program `minergate` is no longer developed.
should we keep it? there is also `minergate-cli` with an equal comit history: initial commit from 2018 and the rest are treewide changes.

Reply:
lets merge it first and then discuss what to do about `minergate`

Reply:
what are we going to do about the `minergate` package?

Reply:
The URL should be changed to web.archive.org.

Reply:
the problem is: `archive.org` checks every file for viruses when uploading. some anti-virus scans detect mining software as viruses. archive.org then blocks the access to these files. so uploading them to archive.org is not really an option.

Reply:
it leaves an impression of being abandoned. I've created https://github.com/NixOS/nixpkgs/issues/133127

Reply:
I am talking about web.archive.org, not archive.org. They are different services.